src/redist/targets/Signing.targets

```diff
@@ -225,16 +160,14 @@
     <!-- Sign engine-->
     <ItemGroup>
       <EngineFileToSign Include="$(CombinedFrameworkSdkHostBundleEngineName)" />
-      <EngineFileSignInfo Include="$([System.IO.Path]::GetFileName('$(CombinedFrameworkSdkHostBundleEngineName)'))"
-                          CertificateName="$(InternalCertificateId)"/>
     </ItemGroup>
     <Microsoft.DotNet.SignTool.SignToolTask
         DryRun="$(_DryRun)"
         TestSign="$(_TestSign)"
         CertificatesSignInfo="$(CertificatesSignInfo)"
```

Comment:
Should this be `CertificatesSignInfo=@(CertificatesSignInfo)`?

Reply:
Ahh yeah. Good catch. May not have made a difference in this repo.